airflow/providers/plexus/CHANGELOG.rst

```diff
@@ -27,6 +27,17 @@
 Changelog
 ---------
 
+3.4.1
+.....
+
```

Comment:
We should add note that this is the last provider release
Similar to what we did for daskexecutor https://github.com/eladkal/airflow/blob/1ed6ae498e3aa8937c171b0f5f1fc3de8f742227/airflow/providers/daskexecutor/CHANGELOG.rst

Reply:
Yep. Fixed.